### PR TITLE
do not throw if value is null

### LINKED ISSUE #11: prop.get(user, 'name.first') will throw if name is null
Seems weird to me that this library throws on things like: `null`, `false`, `3`, `"hi"`.

I feel like the `obj === undefined` check should be `isObj(obj)` or something.

### PR DESCRIPTION
**Pull request by stevemao**
Fixes #11.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Sorry, totally forgot about this one... I've changed my mind, I no longer think it makes sense to throw on accessor. Could you fix the merge conflict and only handle the other cases?

**Comment by sindresorhus** [comment]
ping @stevemao

**Comment by stevemao** [comment]
Hm... I was waiting for #17 to be merged first... Since I think this one is a bit tricker... I'll work on this if you think its better :)

**Comment by stevemao** [comment]
What do you think @SamVerschueren?

**Comment by SamVerschueren** [comment]
Sorry, think I got lost somewhere in the conversation :). Where do you want my opinion on? To merge #17 first and after that this PR?

**Comment by stevemao** [comment]
They all need to be merged. Just need +1s on these PRs

File: index.js [hunk 1]
```diff
@@ -11,7 +11,11 @@ module.exports.get = function (obj, path) {
 	for (var i = 0; i < pathArr.length; i++) {
 		obj = obj[pathArr[i]];
 
-		if (obj === undefined) {
+		if (obj === undefined || obj === null) {
+			if (i !== pathArr.length - 1) {
```
Comment (sindresorhus) [inline index.js]: Can you add a comment on why this is here? I'm a bit unsure what it's doing.

**Comment by stevemao** [comment]
@sindresorhus I have added some comments.

**Comment by SamVerschueren** [comment]
LGTM

**Comment by stevemao** [comment]
Sorry guys... I screwed up with git. Can you please re-push the latest commits to master branch? Or is there a way for me to restore them (I don't have those commits in my local repo)? Sorry!!

**Comment by sindresorhus** [comment]
No worries. Fixed. Btw, you can always use `git reflog` to undo stuff.

**Comment by sindresorhus** [comment]
Thanks @stevemao. Glad we could finally get this merged :)

**Pull request merged**